.eslintrc.js

```diff
@@ -2,10 +2,337 @@
+    "no-unused-expressions": "error",
+    "no-useless-call": "error",
+    "no-useless-concat": "error",
+    "no-useless-return": "error",
+    "no-void": "error",
+    "prefer-named-capture-group": "error",
+    "prefer-promise-reject-errors": "error",
+    radix: "error",
+    "require-unicode-regexp": "error",
+    "vars-on-top": "error",
+    "wrap-iife": "error",
+    yoda: "error",
+    "default-param-last": "error",
+    "grouped-accessor-pairs": ["error", "getBeforeSet"],
+    "no-constructor-return": "error",
+    "prefer-regex-literals": "error",
+
+    "class-methods-use-this": "off",
+    "consistent-return": "off",
+    "max-classes-per-file": "off",
+    "no-alert": "off",
+    "no-empty-function": "off",
+    "no-invalid-this": "off",
+    "no-magic-numbers": "off",
+    "no-param-reassign": "off",
+    "no-return-assign": "off",
+    "no-warning-comments": "off",
+    "require-await": "off",
+
+    // https://github.com/Exelord/eslint-plugin-zero-config/blob/master/lib/configs/-private/eslint/es6.js
+    "arrow-body-style": ["error", "as-needed"],
+    "arrow-parens": "error",
+    "arrow-spacing": "error",
+    "generator-star-spacing": "error",
+    "no-confusing-arrow": ["error", { allowParens: true }],
+    "no-duplicate-imports": "error",
+    "no-useless-computed-key": "error",
+    "no-useless-constructor": "error",
+    "no-useless-rename": "error",
+    "no-var": "error",
+    "object-shorthand": "error",
+    "prefer-const": "error",
+    "prefer-destructuring": [
+      "error",
+      {
+        VariableDeclarator: {
+          array: true,
+          object: true,
+        },
+        AssignmentExpression: {
+          array: false,
+          object: true,
+        },
+      },
+      {
+        enforceForRenamedProperties: false,
+      },
+    ],
+    "prefer-numeric-literals": "error",
+    "prefer-spread": "error",
+    "prefer-template": "error",
+    "rest-spread-spacing": "error",
+    "symbol-description": "error",
+    "template-curly-spacing": "error",
+    "yield-star-spacing": "error",
+    "no-new-symbol": "error",
+    "no-restricted-exports": "error",
+    "no-restricted-imports": "off",
+    "prefer-arrow-callback": "off",
+    "prefer-rest-params": "off",
+    "sort-imports": "off",
+
+    // https://github.com/Exelord/eslint-plugin-zero-config/blob/master/lib/configs/-private/eslint/possible-errors.js
+    "no-await-in-loop": "error",
+    "no-console": "error",
+    "no-extra-parens": ["error", "all", { enforceForArrowConditionals: false }],
+    "no-template-curly-in-string": "error",
+    "no-extra-semi": "error",
+    "no-inner-declarations": "error",
+    "no-unreachable-loop": "error",
+    "require-atomic-updates": "error",
+
+    // https://github.com/Exelord/eslint-plugin-zero-config/blob/master/lib/configs/-private/eslint/strict-mode.js
+    strict: 'error',
+
+    // https://github.com/Exelord/eslint-plugin-zero-config/blob/master/lib/configs/-private/eslint/stylistic-issues.js
```

Comment:
I don't think `stylistic-issues` is included in the final config. not sure what it's there for, but I believe this section can be removed.

Reply:
Awesome! Just removed those rules